frontend/src/components/dashboard/MailList.tsx

```diff
@@ -0,0 +1,26 @@
+import { Grid } from "@material-ui/core"
+import * as React from "react"
+import { request } from "../../utils/fetch"
+import MailItem, { MailListItem } from "./MailItem"
+
+export default function MailList(): JSX.Element {
+  const [mail, setMail] = React.useState<MailListItem[]>([])
+
+  React.useEffect(() => {
+    request<MailListItem[]>("/api/profiles/", "get", true).then((mailList) => {
+      setMail(mailList.parsedBody ? mailList.parsedBody : [])
```

Comment:
Same null coalescing suggestion stated in an earlier comment. I think your recent changes to parsed body might make this a non-issue. 
```suggestion
      setMail(mailList.parsedBody ?? [])
```

Reply:
👍